components/insight/SRC/org/openmicroscopy/shoola/agents/fsimporter/chooser/ImportDialog.java

```diff
@@ -1575,12 +1575,15 @@ public void addImageJFiles(List<FileObject> list,
             FileObject f, ff;
             if (active) {
                 ImagePlus p = WindowManager.getCurrentImage();
+                if (p == null) {
+                    IJ.log("Please open an image first.");
+                    return;
+                }
                 f = new FileObject(p);
-                int id = p.getID();
                 //check if there are associated files
                 int[] values = WindowManager.getIDList();
                 String path = f.getAbsolutePath();
-                if (path != null) {
+                if (path != null && values != null) {
```

Comment:
Can there b a case of a ImageWindow being available and the values being null (i.e the image not having an id?)